.pipelines/pipeline.yaml

```diff
@@ -172,7 +172,7 @@ stages:
       - template: e2e-job-template.yaml
         parameters:
           name: "ubuntu_16_04_linux_e2e"
-          pipelineBuildImage: "containernetworking/pipeline-ci:1.0.6"
+          pipelineBuildImage: "$(BUILD_IMAGE)"
```

Comment:
where are we setting BUILD_IMAGE?

Reply:
set at pipeline runtime so it's more dynamic